### ISSUE TITLE
Add Venmo integration

### ISSUE THREAD

**Issue by josephlou5** [josephlou5/split-the-bill#2]
Somehow allow users to connect their Venmo account or be able to pull up their Venmo username / QR code to make it easier for them to get paid. Bonus points if support is also added to Venmo request all participants for the proper amount, which would actually make this tool a lot more useful.

I haven't used Venmo groups but maybe that could be worth looking into.

Requested by Johnny Dang.

**Comment by josephlou5** [comment]
For reference, Venmo API: https://venmo.com/docs/overview/

But unfortunately, looks like they've retired their Developer API (see [SO post](https://stackoverflow.com/questions/45616742/cant-get-access-token-for-my-venmo-acount), and also the message at the top of every page in their API docs). Seems like it is no longer possible for a regular user to get an access token from their account, and thus no way to properly use their API.

Looks like you could become a business account to get access to their business API (?), but that's infeasible as the goal of this integration would be to make requests on behalf of individual users, and asking them to do additional work on their Venmo account seems like too much.

Oh well. This would've been cool.

**Comment by josephlou5** [comment]
Also considered maybe just generating the Venmo QR code given the user's username or something? Unfortunately I decoded mine and it uses a (surprisingly very long) user ID value, along with a created timestamp (for some reason). So that doesn't really seem feasible either, since there doesn't seem to be an easy way to find your user ID from the app.

I tried going to venmo.com/{username}, and in the page source there does seem to be references to the user ID. However it's buried deep within a `script#__NEXT_DATA__` element. It seems like too much effort for my webpage to fetch a Venmo page, scrape it for some user ID string, and then build a QR code. So this idea is scrapped as well.
